code-challenges.test.js

```diff
@@ -22,7 +22,63 @@
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
+describe("fibSequence", () => {
+    it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+      expect(fibSequence(6)).toEqual([1, 1, 2, 3, 5, 8])
+      expect(fibSequence(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+  })
+
+//   FAIL  ./code-challenges.test.js
+//   fibSequence
+//     ✕ takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence (1 ms)
+
+//   ● fibSequence › takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence
+
+//     ReferenceError: fibSequence is not defined
+
+
+
+
 // b) Create the function that makes the test pass.
+// PSEUDO:
+// Create a function takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence
+// paramter will be number
+// input:
+
+// input:
+// declare a new variable called newFibArr that will return the end result
+// use conditionals for is the number is greater than 2
+// use .push to keep pushing the previous number to be added to the new
+// output: an array that length containing the numbers of the Fibonacci sequence
+// had to google fibonacci sequence
+// syntax errors keep happening on lines 61, 62, and 63
+// I know on line 64 I need to use .push and I need to add together two instances, but I can't figure out how to set that up properly
+// I ended up reaching out to some of my cohort mates to understand how to better set up my function. I understand the pieces of the puzzle, but really struggle with putting them together to make the big picture work. I don't think I would have figured this out on my own. I need to study for loops more.
+
```

Comment:
Great description of your process. I think using your cohortmates is a great approach.